src/Workspaces/Core/MSBuild/MSBuild/MSBuildProjectLoader.cs

```diff
@@ -144,7 +144,9 @@ private DiagnosticReportingMode GetReportingModeForUnrecognizedProjects()
         public async Task<SolutionInfo> LoadSolutionInfoAsync(
             string solutionFilePath,
             IProgress<ProjectLoadProgress>? progress = null,
+#pragma warning disable IDE0060 // TODO: decide what to do with this unusued ILogger, since we can't reliabily use it if we're sending builds out of proc
```

Comment:
couldn't hte OOP work log to its own logger that forwards to this?

Reply:
@CyrusNajmabadi: yes, we could. I don't want to do that engineering work unless we have some belief we need to do it though.